aware-core/src/main/java/com/aware/Aware_Preferences.java

```diff
@@ -465,6 +465,16 @@
     public static final String FREQUENCY_WEBSERVICE = "frequency_webservice";
 
     /**
+     * AWARE webservice only: remove local data always
+     */
+    public static final String WEBSERVICE_ONLY = "webservice_only";
```

Comment:
Maybe rename this to: WEBSERVICE_NO_LOG_HISTORY ? Webservice only reads to me as no local storage and constantly sending data remotely in real-time, which is not the case.